pages/_category/browser/_id.vue

```diff
@@ -0,0 +1,15 @@
+<template>
```

Comment:
The file and folders names with the `_`... is this something that nuxt requires?

Reply:
Yes, these are dynamic routes. Look at the PR description to see what this structure generates.